.travis.yml

```diff
@@ -22,10 +18,9 @@ branches:
     - master
 
 script:
-  - npm run lint:hbs
-  - npm run lint:js
-  - npm run test:coverage
+  - yarn ci
 
-after_script:
-  - npm run coveralls
-  - npm run codacy-coverage
+# @TODO Turn back after coveralls and codacy implementation are fixed
+#after_script:
+#  - yarn coveralls
+#  - yarn codacy-coverage
```

Comment:
Do the `coveralls` and `codacy` implementations need to be fixed on our side? If so, let's add an issue once we merge this PR. Thanks.

Reply:
Ticket is opened, so we can investigate. https://github.com/zoltan-nz/library-app/issues/66